man/systemd.network.xml

```diff
@@ -1903,6 +1903,16 @@
         <varname>DNS=</varname>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LogServers</varname></term>
+
+        <listitem><para>Specifies a list of MIT-LCS UDP servers available to the client.
```

Comment:
Dunno, it's still not obvious what this is used for based on this description, but OK.